workspaces/ui-v2/tsconfig.json

```diff
@@ -4,6 +4,10 @@
     "jsx": "react",
     "outDir": "build",
     "resolveJsonModule": true,
-    "rootDir": "src"
+    "rootDir": "src",
+    "baseUrl": ".",
```

Comment:
I could only get this to work when i specified _both_ the tsconfig + webpack resolve.alias fields. I'm not sure why we need to double specify this and it'll be confusing since we'll need to add this in two locations if we want to update stuff???